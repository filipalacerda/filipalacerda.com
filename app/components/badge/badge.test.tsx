import { render, screen } from "@testing-library/react";

import "@testing-library/jest-dom";

import Badge from ".";

describe(Badge.name, () => {
  let container: HTMLElement;

  beforeEach(() => {
    container = render(<Badge name="React" backgroundColor="" />).container;
  });

  it("should match snapshot", () => {
    expect(container).toMatchSnapshot();
  });

  it("renders the provided name", () => {
    expect(screen.getByText("React")).toBeInTheDocument();
  });
});
