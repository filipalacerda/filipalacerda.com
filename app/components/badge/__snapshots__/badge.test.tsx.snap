// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should match snapshot 1`] = `
<div>
  <div
    class="rounded-lg text-xs w-fit p-1"
    style="color: white;"
  >
    React
  </div>
</div>
`;
